src/assets/mocks/contacts.ts

```diff
@@ -0,0 +1,33 @@
+export interface Contact {
+  img: string;
+  title: string;
+  description_1: string;
+  description_2: string;
+}
+
+export const CONTACTS: Contact[] = [
+  {
+    img: 'assets/images/call.svg',
```

Comment:
Why are you providing icons in mocks?

Reply:
I changed mock to const, moved them into contacts folder and replaced each img src with src from third-party github.